### PR TITLE
doc/Install.md: Troubleshooting: loadavg, filesystem ACL, big uploads

### PR DESCRIPTION
**Pull request by bkil**
Relates in part to #10131

### CODE REVIEW

File: doc/Install.md [hunk 1]
```diff
@@ -449,3 +449,58 @@ section:
...
+You should instead enact limits in your web server configuration based on the number of entry processes to cap the concurrent memory usage of your PHP processes.
+See `RLimitMEM`, `RLimitCPU`, `RLimitNPROC`, `StartServers`, `ServerLimit`, `MaxRequestsPerChild`, `pm.max_children`, `pm.start_servers` and related options in your server.
+
+### Error uploading even small image files
+
+You tried to upload an image up to 100kB and it failed.
+
+You may not have the ownership or file mode set correctly if you are using the file system storage backend.
+
+Change the backend to database. If this solves it, that that is was needs to be fixed.
```
Comment (MrPetovan) [inline doc/Install.md]:
```suggestion
Change the backend to database. If this solves it, that is what needs to be fixed.
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: doc/Install.md (hunk 1 above)
Comment (bkil) [reply doc/Install.md]: Thanks, fixed in https://github.com/friendica/friendica/pull/11277/commits/dac67baf1104c5ebc11e45b560efb4b2fd69a0c9.

**Review by MrPetovan — APPROVED** [review]
Thank you for your work!

File: doc/Install.md [hunk 2]
```diff
@@ -449,3 +449,58 @@ section:
 	sql_mode = '';
 
 After that, restart mysql and try again.
+
+### Your worker never or rarely runs
+
+Friendica is coded to always play nice. It checks whether the host machine is idle enough and if it _seems_ to be overloaded, it intermittently refuses to process the worker queue.
```
Comment (tobiasd) [inline doc/Install.md:455]: please use one sentence per line for your next changes to the documentations...

Comment (bkil) [reply doc/Install.md:455]: Good idea! Incidentally, I usually do that nowadays in my own docs as well. I'll fix this with the next additions. I hope to submit a few more frequently discussed ones from our chat.

**Pull request merged**